prow/github/types.go

```diff
@@ -55,6 +55,13 @@ const (
 	stateCannotBeChangedMessagePrefix = "state cannot be changed."
 )
 
+// Possible types of merges for the GitHub merge API
+const (
```

Comment:
nit: Looks like the constants above don't do this but looking below type aliases help as well with these lists of values:

```go
type PullRequestEventAction string
```
That way we're not stringly typed and it's obvious when someone needs a `MergeType`

Reply:
Updated